jwst/refpix/reference_pixels.py

```diff
@@ -1327,6 +1365,16 @@ def do_left_right_correction(self, group, refvalues):
         return
 
     def do_corrections(self):
+        if self.is_subarray:
+            self.do_subarray_corrections()
+        else:
+            self.do_fullframe_corrections()
+
+    def do_subarray_corrections(self):
+        log.warning("Refpix correction skipped for MIRI subarray")
+        return
```

Comment:
Does this in turn trigger the setting of the step completion status to "SKIPPED"?

Reply:
Actually, this case is handled higher up now, so this line should never be needed.  The completion status will indeed be set to skipped for MIRI subarrays.

Reply:
👍